README.md

```diff
@@ -19,16 +28,60 @@ use {
   requires = {
     'nvim-lua/plenary.nvim',
     'nvim-treesitter/nvim-treesitter'
-    },
-  config = function() -- optional
-    require("ruby_nvim").setup({
-      test_cmd = "ruby",  -- the default value
-      test_args = {},  -- the default value
-    })
-  end,
+  }
 }
 ```
 
-# Disclaimer
+Example with [vim-plug](https://github.com/junegunn/vim-plug):
+
+```vim
+Plug 'nvim-lua/plenary.nvim'
+Plug 'nvim-treesitter/nvim-treesitter'
+Plug 'vinibispo/ruby.nvim'
+```
+
+---
+
+# Setup
+
+## Default config
+
+Ruby.nvim comes with sane defaults so that you can get going without having to add much to your config
+
+```lua
+local ruby_nvim = require("ruby_nvim")
+
+ruby_nvim.setup({
+  test_cmd = "ruby" -- command that :RubyTest is gonna run,
+  test_args = {} -- args that :RubyTest is gonna run
+})
+```
+
+---
+
+# Commands
```

Comment:
100% a matter of style and personal preference, but I think this structure involving three levels of headings (_Commands, Run, :RubyRun_, for example) makes it difficult to read and to get an overall of what the plugin offers.

I tend to prefer a concise table, as in `yaml.nvim`:

> | Command | Description |
> |:--|:--|
> | `:YAMLView` | Shows the full path and value of the current key/value pair |
> | `:YAMLYank [register]` | Yanks the full path and value of the current key/value pair. The default register is the unnamed one (`"`) |
> | `:YAMLYankKey [register]` | Yanks the full path of the key for the current key/value pair. The default register is the unnamed one (`"`) |
> | `:YAMLYankValue [regster]` | Yanks the value of the current key/value pair. The default register is the unnamed one (`"`) |
> | `:YAMLQuickfix` | Generates a quickfix with key/value pairs |
> | `:YAMLTelescope` | Full path key/value fuzzy finder via [Telescope](https://github.com/nvim-telescope/telescope.nvim) **if installed** |

Reply:
I'll fix it